pyroclast/common/preprocessed_dataset.py

```diff
@@ -0,0 +1,30 @@
+import tensorflow as tf
+
+
+class PreprocessedDataset(object):
```

Comment:
This class definitely needs testing before we merge.

Reply:
Also, add a docstring or something that talks about what kind of preprocessing this class does. Maybe we should rename the class because it is not all that descriptive.